config/G18_02a/ModelConfiguration.xml

```diff
@@ -138,7 +138,7 @@ STFC, Rutherford Appleton Laboratory
   <param type="alg" name="XSecModel@genie::EventGenerator/NucleonDecay">   genie::DummyPXSec/Default             </param>
   <param type="alg" name="XSecModel@genie::EventGenerator/NNBarOsc">       genie::NNBarOscDummyPXSec/Default     </param>
 
-
+  <param type="alg" name="XSecModel@genie::EventGenerator/EventLibraryInterface">  genie::vmc::EvtLibPXSec/Default   </param>
```

Comment:
I admit to being a little lost in all the config files and tunes.
Ideally this would be the "gibuu external v whatever" tune. And then we could have have the "nuwro external v something else" tune, with the same config, just a different library configured. That would presumably prevent the spline xmls getting mixed up.

Reply:
Done, as per discussion on slack.
Use --tune EX00_00a_00_000 to run.